tests/FunctionalTest.php

```diff
@@ -2,7 +2,6 @@
 
 namespace Clue\Tests\React\HttpProxy;
 
-use Clue\React\Block;
 use Clue\React\HttpProxy\ProxyConnector;
 use React\EventLoop\Loop;
```

Comment:
I think we can also get rid of this statement as it's not used anymore in this class

Reply:
I meant:
```php
use React\EventLoop\Loop;
```

Reply:
Fixed thanks for pointing out 👍🏼